src/main/java/io/github/moulberry/notenoughupdates/mixins/MixinOFDynamicLights.java

```diff
@@ -32,15 +31,9 @@
 @Mixin(targets = "net.optifine.DynamicLights", remap = false)
 public class MixinOFDynamicLights {
 
-	@Inject(method = "getLightLevel(Lnet/minecraft/item/ItemStack;)I", at = @At("TAIL"))
-	private static int getLightLevel(ItemStack itemStack, CallbackInfoReturnable<Integer> cir) {
-		String internalName = DynamicLightItemsEditor.Companion.resolveInternalName(itemStack);
-		if (internalName == null) return 0;
-
-		if (NotEnoughUpdates.INSTANCE.config.hidden.dynamicLightItems.contains(internalName)) {
-			return 15;
-		}
-		return 0;
+	@Inject(method = "getLightLevel(Lnet/minecraft/item/ItemStack;)I", at = @At("TAIL"), cancellable = true)
+	private static void getLightLevel(ItemStack itemStack, CallbackInfoReturnable<Integer> cir) {
+		cir.setReturnValue(DynamicLightItemsEditor.findDynamicLightItems(itemStack));
```

Comment:
Can you only set the return value if we have an actual override set? That way we don't block other mods from doing similar things.